packages/ruby/src/index.ts

```diff
@@ -59,16 +60,30 @@ async function bundleInstall(
       gemfileContent.replace('ruby "~> 2.7.x"', 'ruby "~> 2.7.0"')
```

Comment:
do we need to change this line for 3.2?

Reply:
<!-- Questions comments seek clarification.  -->
**question (blocking):** ‏do we need to adjust this?

Reply:
49e93d3c744225f0f66322aea360a6725687cd42